.github/workflows/release.yml

```diff
@@ -0,0 +1,176 @@
+name: Release
+
+on:
+  push:
+    tags:
+      - "v*.*.*"
+
+jobs:
+  build-linux:
+    name: Build Linux
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
```

Comment:
Worth noting that these "actions" can run any code they want and nothing is stopping them from updating the tag to another commit. I generally try not to trust them for any software I'm releasing and instead use a specific commit hash and add a comment saying which version it is